lib/general.js

```diff
@@ -54,9 +54,11 @@ class OptionalOpencv4nodejsCommandCheck extends DoctorCheck {
     } catch (err) {
       log.debug(err);
     }
-    return stdout.includes(paclageName)
-      ? okOptional(`${paclageName} is installed.`)
-      : nokOptional(`${paclageName} cannot be found.`);
+    if (stdout.includes(paclageName)) {
+      const eachLine = stdout.split(EOL);
+      return okOptional(`${paclageName} is installed at: ${eachLine[0]}. Installed version is: ${eachLine[1].replace('└── opencv4nodejs@', '')}`);
```

Comment:
Please verify first if the array length is ok to avoid range error

Reply:
I've changed to get the version with `pop()`